tools/deployment/make_osx_package.sh

```diff
@@ -11,7 +11,12 @@ set -e
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 SOURCE_DIR="$SCRIPT_DIR/../.."
-BUILD_DIR="$SOURCE_DIR/build/darwin"
+BUILD_VERSION=`sw_vers -productVersion | awk -F '.' '{print $1 "." $2}'`
```

Comment:
Do it like this: https://github.com/facebook/osquery/blob/master/tools/build.sh#L15

Such that we're consistent.